Templates/Kotlin/GraphApi.stencil

```diff
@@ -15,15 +15,25 @@ import java.lang.reflect.Type
 
 const val DEFINED_NULL_FLAG = "__DEFINED_NULL"
 
-interface SyrupOperation<T: Response> {
-    fun getQueryString(): String
+interface SyrupOperation<T : Response> {
+    val rawQueryString: String
+    fun getQueryString(): String {
```

Comment:
We should think of a more descriptive name to differentiate the two. Not really sure the difference between `getQueryString` and `rawQueryString` when they both return `String`.

Reply:
rawQueryString and networkQueryString?

Reply:
Maybe make it private and rename it `operationTemplate`, and `rawQueryString` -> `toSerializedOperation` or just `operation`. 

Misnomer using the term `query` here.

Reply:
decided not to include these changes in this PR to avoid excessive refactoring